### DISCUSSION TITLE
Features request

### DISCUSSION THREAD

**Discussion by gpietro** [directus/directus#13477] [Feature Requests]
- **Documentation** could be improved by adding more examples and in depth usage; especially on how to use extensions
- Improve dev experience with **debugging** (extensions) and **unit testing**
- Possibility to store **configurations** in a **file** instead of database. Easier to track&merge changes than migrations.
- **+** it would be great and a game changer to have the possibility to write custom modules in **React.js**

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
